OTAnalytics/plugin_ui/line_section.py

```diff
@@ -236,10 +245,15 @@ def _create_section(self) -> None:
         if self._metadata == {}:
             raise ValueError("Metadata of line_section are not defined")
         name = self._metadata[ID]
+        relative_offset_coordinates_enter = self._metadata[RELATIVE_OFFSET_COORDINATES][
+            "section-enter"
```

Comment:
@briemla I think we need a constant here: Where should we define it? Can we take it from the enum class in types.py or do we have to define the constants somewhere else?